docs/user/reference/metrics/event.md

```diff
@@ -1,39 +1,22 @@
 # Events
 
-Events allow recording of e.g. individual occurrences of user actions, say every time a view was open and from where.
+Events allow recording of e.g. individual occurrences of user actions,
+say every time a view was open and from where.
```

Comment:
Not blocking, just out of curiosity: why are we splitting this line?

Reply:
No reason other than I disliked this line being so long, so I broke it at a reasonable point (also see [semantic line breaks](https://sembr.org/))